ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -1057,16 +1061,15 @@
 
 
 (defn detect-route-changes-for-service-old [db {:keys [start-date service-id] :as route-query-params}]
-  (let [type (db-route-detection-type db service-id)
```

Comment:
type is still used on row 1066

Reply:
Thank you! Really good catch.
Added type back and renamed it to route-hash-id-type to avoid similar problems.